src/olympia/accounts/views.py

```diff
@@ -455,7 +455,8 @@ def destroy(self, request, *args, **kwargs):
             logout_user(request, response)
         return response
 
-    @detail_route(
+    @action(
+        detail=True,
         methods=['delete'], permission_classes=[
             AnyOf(AllowSelf, GroupPermission(amo.permissions.USERS_EDIT))])
```

Comment:
It feels like there should be a way of formatting this that's either more compact... or consistent with line breaks.  But meh.  Nit, no action needed.